test/parallel/test-stdin-script-child.js

```diff
@@ -3,30 +3,32 @@ const common = require('../common');
 const assert = require('assert');
 
 const spawn = require('child_process').spawn;
-const child = spawn(process.execPath, [], {
-  env: Object.assign(process.env, {
-    NODE_DEBUG: process.argv[2]
-  })
-});
-const wanted = `${child.pid}\n`;
-let found = '';
+for (const args of [[], ['-']]) {
+  const child = spawn(process.execPath, args, {
+    env: Object.assign(process.env, {
```

Comment:
🤔 this is a super weird construct. IMHO If it's to suss out a bug it should be near https://github.com/nodejs/node/blob/master/test/parallel/test-child-process-spawn-shell.js#L51
Personaly I'd rather see this fixed here, but I won't block for it.